containers/hydra/Dockerfile

```diff
@@ -43,10 +43,13 @@ RUN /setup/install-sbomnix.sh
 
 # copy all other setup scripts that will be used later
 COPY postgres.sh hydra.sh populate.sh packages.lst postbuild.py cachecopy.sh \
-    schedule.sh messager.py provenance.sh provenance.py sha256tree.py \
+    schedule.sh messager.py provenance.sh provenance.py \
     compress_outputs.sh webupload.sh \
     /setup/
 
+# Fetch sha256tree.py from repo
+RUN wget -O /setup/sha256tree.py https://raw.githubusercontent.com/tiiuae/ci-public/main/sha256tree/sha256tree.py
```

Comment:
Pin the commit instead of using whatever is the HEAD of main branch.

Reply:
Ok. Needed another commit for this.